packages/open-next/src/overrides/wrappers/cloudflare.ts

```diff
@@ -65,4 +65,7 @@ export default {
   name: "cloudflare",
   supportStreaming: true,
   edgeRuntime: true,
-};
+} satisfies Wrapper<
+  InternalEvent,
+  InternalResult | ({ type: "middleware" } & MiddlewareOutputEvent)
```

Comment:
We should probably add `type:middleware` to the `MiddlewareOutputEvent` directly